iOS/Scripts/add_secrets_if_there.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+SOURCE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+FILE="$SOURCE_DIR/add_secrets_to_env.sh"
+
+echo "FILE: $FILE"
+
+if [ -f $FILE ]; then
+  echo "Environment exists, running..."
+
+  # Get the secrets into the environment
+  source "$FILE"
+else
+  echo "File does not exist - hope you added things in Circle's private keys!"
+fi
+
+PARENT_DIR=$(DIRNAME $SOURCE_DIR)
+
+echo "PARENT $PARENT_DIR"
+FIREBASE_PLIST="$PARENT_DIR/DoorbellBotNative/GoogleService-Info.plist"
+
+echo "FIREBASE_PLIST $FIREBASE_PLIST"
+
+/usr/libexec/Plistbuddy -c "Set :AD_UNIT_ID_FOR_BANNER_TEST $FIR_AD_UNIT_ID_FOR_BANNER_TEST" "$FIREBASE_PLIST"
```

Comment:
I know this is spectacularly repetitive - I'm working on pulling this out into something I'm doing with Swift scripts in another branch.

Reply:
I think it would be possible to store the secrets in a properties-file read by Gradle and expose them through the shared code actually. Do you think that could be a solution later at some point or what do you think?

Reply:
yeah i was going to try and get it running through known processes on iOS and Android and then see what could be combined